crates/stc_ts_type_checker/tests/conformance/classes/members/privateNames/privateNamesInGenericClasses.ts.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
-    required_error: 0,
-    matched_error: 5,
-    extra_error: 9,
-    panic: 0,
+    required_error: 5,
+    matched_error: 0,
+    extra_error: 0,
+    panic: 1,
```

Comment:
Can you investigate this?

Reply:
The panic is caused by this line https://github.com/dudykr/stc/blob/a0df88a1cfaeccdf444c87b2a70b20a72301707d/crates/stc_ts_types/src/lib.rs#L449. How do I show that a `Type` is private? Can I add something like `is_private` to the type metadata?

Reply:
You should check for `Key::Private` before calling `key.ty()`